docs/quality-assurance/README.md

```diff
@@ -10,3 +10,5 @@ These documents detail how we work as a Test Community at UKHO. We aim to provid
   * The guidelines for using Test Automation
 * [Our Test Standards](test-code-standards.md)
   * How we plan, write and organise our tests
+  [Our Test Standards](test-policycode.md)
```

Comment:
You are missing the asterisk on this line

Reply:
```suggestion
 * [Our Test Standards](test-policycode.md)
```